scripts/compileAjvValidators.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
```

Comment:
To avoid a new pattern, can this be moved to a top-level file called `ajv.config.js`?

Reply:
so that eslint won't check it? idk i think that would be a little bit of misnaming because it's not a config file

Reply:
not for eslint, just to avoid more complexity in the directory tree by adding a new `scripts` folder. What about just `ajv.js`?